integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoQueryTestTool.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.trino;
+
+import com.datastrato.gravitino.integration.test.util.ITUtils;
+import java.io.File;
+import java.util.Arrays;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.PosixParser;
+import org.apache.logging.log4j.util.Strings;
+
+public class TrinoQueryTestTool {
```

Comment:
Will this tool run in the CI pipeline?

Reply:
No, it used for QA, or debug with the tester or test sets.